### DISCUSSION TITLE
Header - customisable

### DISCUSSION THREAD

**Discussion by manalishi79** [UKHomeOffice/design-system#368] [Ideas]
For internal services we often need a way to make it branded for that specific portfolio (PPPT for example)

Some options and customizable areas would be helpful to extend the HO header.

For example, replacing the HO logo and title with a service name

**Comment by karypun** [comment]
@manalishi79 let's discuss this as it would be good to understand this a bit more and see examples of what you mean

**Comment by karypun** [comment]
@manalishi79 I've added this to the proposed column for further discussion with the wider working group.

**Comment by manalishi79** [comment]
If not possible/easy to build out a whole customisable header could an easier process be to setup a way to allow designers "edit" chunks on the template like on GOV.uk prototype kit
